core/src/engine/Method_GNEB.cpp

```diff
@@ -274,26 +274,31 @@ namespace Engine
 
 			std::string preChainFile;
 			std::string preEnergiesFile;
-			if (this->systems[0]->llg_parameters->output_tag_time)
-			{
-				preChainFile = this->parameters->output_folder + "/" + starttime + "_Chain";
-				preEnergiesFile = this->parameters->output_folder + "/" + starttime + "_Chain_Energies";
-			}
-			else
-			{
-				preChainFile = this->parameters->output_folder + "/Chain";
-				preEnergiesFile = this->parameters->output_folder + "/Chain_Energies";
-			}
+            std::string fileTag;
+            
+            if (this->systems[0]->llg_parameters->output_file_tag == "<time>")
```

Comment:
As the tag has been abstracted as a parameter across all `Method`s, it would make sense to get this from a function.
However, it's important to use the child class `Parameters` member, (i.e. `gneb_parameters` in the GNEB case)!
At least the latter should be fixed in this PR.